src/resources/gen/events.ts

```diff
@@ -0,0 +1,286 @@
+/// <reference path="../interfaces.ts" />
+
+/**
+ * This file is auto-generated by the `asana-api-meta` NodeJS package.
+ * We try to keep the generated code pretty clean but there will be lint
+ * errors that are just not worth fixing (like unused requires).
+ * TODO: maybe we can just disable those specifically and keep this code
+ * pretty lint-free too!
+ */
+/* tslint:disable */
+var resource = <Resource>{
+  "templates": [
```

Comment:
You can probably delete the "templates" key from the generated json? It's only used to make substitutions in the rest of the structure.